openfisca_uk/parameters/consumption/carbon/consumption.yaml

```diff
@@ -0,0 +1,13 @@
+description: Consumption emissions, [more description]
```

Comment:
Could you replace [more description] with something descriptive about this data?

Reply:
Done. Pls check

Reply:
I see it now says `household`, but how about: "Total household consumption-based carbon emissions"?

Reply:
Okay ,can we change it to [Carbon emissions] ,[household based consumption]?
I have changed it. Pls check.

Reply:
Not sure why we need the square brackets? I think my suggestion is more in line with other parameter descriptions if that's OK.

Reply:
Square brackets are for  explanatory purpose. Not adding it though.